packages/metrics/test/unit/calculateMetrics.spec.ts

```diff
@@ -195,7 +195,7 @@ describe(calculateMutationTestMetrics.name, () => {
     expect(mutant1?.killedByTests).deep.eq([test2]);
     expect(mutant1?.coveredByTests).deep.eq([test1, test2, test3]);
     expect(mutant2?.killedByTests).undefined;
-    expect(mutant2?.coveredByTests).deep.eq([test1, test4]);
+    expect(mutant2?.coveredByTests).deep.eq([test4, test1]);
```

Comment:
`coveredBy` on this mutant is actually `coveredBy: ['test-4', 'test-1']` so `test4` being first actually makes more sense to me here, I'm not sure why it was reversed beforehand